packages/calcite-components/src/components/table/table.stories.ts

```diff
@@ -1065,6 +1065,80 @@ export const tableCellCssBackgroundVariable_TestOnly = (): string =>
     </calcite-table-row>
   </calcite-table>`;
 
+export const tableBorderedWithStripedAndSingleFooter_TestOnly = (): string =>
+  html`<calcite-table striped bordered caption="Simple-bordered-with-footer table" dir="rtl" selection-mode="multiple">
```

Comment:
should this have dir=rtl? its not in the name of the test.

Reply:
Removed, thanks for catching.